<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>getImage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>821</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string>imageFiles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>loadPNG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoadPLIST">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>180</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>loadPLIST</string>
    </property>
   </widget>
   <widget class="QLabel" name="savePixLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>260</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pixNameLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>40</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>png name</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="plistNameLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>210</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>plist name</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="getPixButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>getPixtures</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputPathEdit">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>/Users/fuzhuo/Documents/</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
